src/Avalonia.Controls/Platform/Screen.cs

```diff
@@ -60,6 +60,24 @@ public Screen(double scaling, PixelRect bounds, PixelRect workingArea, bool isPr
             this.Bounds = bounds;
             this.WorkingArea = workingArea;
             this.IsPrimary = isPrimary;
-        } 
+        }
+
+        /// <summary>
+        /// Determines whether the <see cref="Screen"/> is equal to the specified object.
+        /// </summary>
+        /// <param name="o">The object with which to test equality.</param>
+        /// <returns>True if the objects are equal, otherwise false.</returns>
+        public override bool Equals(object? o)
```

Comment:
Also note if you override Equals() you should be getting a warning somewhere telling you to consider overriding GetHashCode() as well.

Personally, if I go this far I also implement IEquatable<T>.